crates/uv-resolver/src/lock/mod.rs

```diff
@@ -2452,7 +2460,7 @@ impl PackageWire {
 #[derive(Clone, Debug, Eq, Hash, PartialEq, PartialOrd, Ord, serde::Deserialize)]
 pub(crate) struct PackageId {
     pub(crate) name: PackageName,
-    pub(crate) version: Version,
+    pub(crate) version: Option<Version>,
```

Comment:
This was a breaking change for those not specifying / bumping a `required-version`: lock gets modified by new uv to omit version for an editable project and then project dev with older uv chokes on the new lock when they pull. Presumably pre-1.0 this is to be expected though. I saw the changelog entry and even noticed the uv lock status fly-by noting a change but did not take this in for what it meant.

Reply:
Yeah sorry, this was an oversight.